core/src/main/java/com/aem/demo/core/services/impl/RssFeedServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.aem.demo.core.services.impl;
+
+import com.aem.demo.core.dto.NewsCard;
+import com.aem.demo.core.services.RssFeedService;
+import com.rometools.rome.feed.synd.SyndEnclosure;
+import com.rometools.rome.feed.synd.SyndFeed;
+import com.rometools.rome.io.FeedException;
+import com.rometools.rome.io.SyndFeedInput;
+import com.rometools.rome.io.XmlReader;
+
+import lombok.extern.slf4j.Slf4j;
+import org.osgi.service.component.annotations.Component;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+@Component(service = RssFeedService.class)
+@Slf4j
+public class RssFeedServiceImpl implements RssFeedService {
+
+    @Override
+    public List<NewsCard> readFeed() {
+        SyndFeedInput input = new SyndFeedInput();
+        SyndFeed feed = null;
+        try {
+            URL feedUrl = new URL("https://www.nasa.gov/rss/dyn/educationnews.rss");
```

Comment:
Please consider creating an OSGi config for the values like this (this can be the next step)